README.md

```diff
@@ -16,32 +16,107 @@ Flagd is a simple command line tool for fetching and presenting feature flags to
 3. Start the process: `./flagd start -f config/samples/example_flags.json --service-provider http --sync-provider filepath`
 
 This now provides an accessible http or [https](#https) endpoint for the flags:
+### Resolve a boolean value
 
+Command:
+```sh
+curl -X POST "localhost:8013/flags/myBoolFlag/resolve/boolean"
 ```
-$ curl -X POST "localhost:8013/flags/myBoolFlag/resolve/boolean"
-// {"value":true,"reason":"STATIC","variant":"on"}
+Result:
+```sh
+{"value":true,"reason":"STATIC","variant":"on"}
+```
+<br />
```

Comment:
Is this required?

Reply:
no, just felt that the spacing looked nice!

happy to remove (or keep it if there is a better way to be doing it)